tests/torch_conv1d_group_test.hpp

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include "load_csv.hpp"
+#include "RTNeural/RTNeural.h"
+
+namespace torch_conv1d_group_test
+{
+template <typename T>
+std::vector<std::vector<T>> loadFile2D(std::ifstream& stream)
+{
+    std::vector<std::vector<T>> vec;
+
+    std::string line;
+    if(stream.is_open()) {
+        while(std::getline(stream, line)) {
+            std::vector<T> lineVec;
+            std::string num;
+            for (auto ch : line) {
+                if (ch == ',') {
+                    lineVec.push_back(static_cast<T>(std::stod(num)));
+                    num.clear();
+                    continue;
+                }
+
+                num.push_back(ch);
+            }
+
+            lineVec.push_back(static_cast<T>(std::stod(num)));
+            vec.push_back(lineVec);
+        }
+
+        stream.close();
+    }
+
+    return RTNeural::torch_helpers::detail::transpose(vec);
+}
+
+template <typename T>
+int testTorchConv1DGroupModel()
+{
+    if (std::is_same<T, float>::value)
+        std::cout << "TESTING TORCH/CONV1D GROUP MODEL WITH DATA TYPE: FLOAT" << std::endl;
+    else
+        std::cout << "TESTING TORCH/CONV1D GROUP MODEL WITH DATA TYPE: DOUBLE" << std::endl;
+
+    const auto model_file = std::string { RTNEURAL_ROOT_DIR } + "models/conv1d_torch_group.json";
+    std::ifstream jsonStream(model_file, std::ifstream::binary);
+
+    nlohmann::json modelJson;
+    jsonStream >> modelJson;
+
+    RTNeural::ModelT<T, 6, 3, RTNeural::Conv1DT<T, 6, 3, 1, 1, 3, false>> model;
+    RTNeural::torch_helpers::loadConv1D<T>(modelJson, "", model.template get<0>());
```

Comment:
Do you know if TensorFlow also has a "groups" concept for their Conv1D layer? That might be a good thing to test for also, although we can add that in a future PR if it turns out to be complicated.

Reply:
I believe so, and it's functionally the same as PyTorch's

Reply:
Okay cool! I can take a look at adding a test case on the Tensorflow side.